homework_1/task_01/fizzBuzz/fizzBuzz.go

```diff
@@ -0,0 +1,36 @@
+package fizzBuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+
+func FizzBuzzPlay(start, end int) ([]string, error) {
+	if start <= 0 {
+		return nil, errors.New("invalid value, range start should be greater than 0")
+	}
+	if end <= 0 {
+		return nil, errors.New("invalid value, range end should be greater than 0")
+	}
+	if start > end {
+		return nil, errors.New("range start is greater than range end")
+	}
+
+
+	var output []string
+
+	for i := start; i <= end; i++ {
+		if i % 15 == 0 {
+			output = append(output, "FizzBuzz")
+		} else if i % 5 == 0 {
+			output = append(output, "Buzz")
+		} else if i % 3 == 0 {
+			output = append(output, "Fizz")
+		} else {
+			output = append(output, strconv.Itoa(i))
+		}
+	}
+	return output, nil
+}
```

Comment:
Make sure to add an empty new line at the end of your files!
https://axilis.slack.com/archives/C020JHMJ274/p1620999827000800?thread_ts=1620815275.002100&cid=C020JHMJ274